pkg/api/meta/pods.go

```diff
@@ -18,12 +18,14 @@ import (
 // resourcesToCheck is a map of resources and corresponding kinds of things that
 // we want handled in this plugin
 var resourcesToCheck = map[unversioned.GroupResource]unversioned.GroupKind{
-	kapi.Resource("pods"):                                       kapi.Kind("Pod"),
-	kapi.Resource("podtemplates"):                               kapi.Kind("PodTemplate"),
-	kapi.Resource("replicationcontrollers"):                     kapi.Kind("ReplicationController"),
-	batch.Resource("jobs"):                                      batch.Kind("Job"),
-	batch.Resource("jobtemplates"):                              batch.Kind("JobTemplate"),
+	kapi.Resource("pods"):                   kapi.Kind("Pod"),
+	kapi.Resource("podtemplates"):           kapi.Kind("PodTemplate"),
+	kapi.Resource("replicationcontrollers"): kapi.Kind("ReplicationController"),
+	batch.Resource("jobs"):                  batch.Kind("Job"),
+	batch.Resource("jobtemplates"):          batch.Kind("JobTemplate"),
+	// TODO do we still need this or is cronjob sufficient?
```

Comment:
I'm just going to add a follow-up comment to sweep the repo for unnecessary refs to scheduled jobs and pet sets.